twitter1.css

```diff
@@ -0,0 +1,299 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+:root {
+  --right-menu-bg-clr: #f5f8fa;
+}
+:root {
+  --accent-clr: #1da1f2;
+}
+:root {
+  --xtr-light-gry: #e1e8ed;
+}
+:root {
+  --light-gry: #aab8c2;
+}
+:root {
+  --dark-gry: #657786;
+}
+:root {
+  --blk-clr: #14171a;
+}
+p {
+  font-size: 22px;
+}
+body {
+  background-color: #fff;
+  font-family: "Roboto", sans-serif;
+}
+.container {
+  border-top: 1px var(--light-gry) solid;
+  width: max-content;
+  width: 60%;
+  height: 100vh;
+  /* margin: auto; */
+  display: grid;
+  grid-template-columns: 400px auto 500px;
+  grid-template-rows: 200px auto;
+  margin-top: 40px;
+  column-gap: 20px;
+}
+
+.left-menu {
+  display: grid;
+  grid-template-rows: 1fr 50px;
+  margin-left: 80px;
+  margin-right: 40px;
+}
+.left-icons-n-list {
+  display: grid;
+  grid-template-columns: 50px auto;
+}
+.lft-btm--e1-b {
+  display: grid;
+  grid-template-columns: 50px 1fr auto;
+  grid-auto-rows: auto;
+}
+.position {
+  margin-top: 550px;
+  padding: 10px;
+}
+.pic-profile {
+  width: 50px;
+  height: 50px;
+  border: ;
+  border-radius: 10px;
+}
+
+.head-n-main {
+  display: grid;
+  grid-auto-rows: auto;
+}
+.header {
+  display: grid;
+  grid-template-columns: 100px auto 100px;
+  grid-template-rows: 50px 50px 100px;
+  /* row-gap: 1.5em; */
+}
+.star {
+  text-align: center;
+}
+.img-header {
+}
+.img-hdr-pfl {
+  width: 50px;
+  border-radius: 50%;
+}
+.whats-h-header {
+  color: var(--dark-gry);
+  font-family: lato;
+  font-weight: bold;
+  font-size: 38px;
+}
+
+.header-btn {
+  border: solid 1px var(--accent-clr);
+  border-radius: 30px;
+  background-color: var(--accent-clr);
+  height: 40px;
+  color: var(--xtr-light-gry);
+  font-size: 16px;
+  font-weight: bold;
+  text-align: center;
+}
+.hdr {
+  margin-bottom: 60px;
+  padding-right: 20px;
+}
+.house {
+  color: var(--accent-clr);
+  width: 30px;
+  height: 30px;
+  padding: 5px;
+  margin-top: 28px;
+}
+.house-2 {
+  color: var(--accent-clr);
+  width: 30px;
+  height: 30px;
+}
+.twitter {
+  color: var(--accent-clr);
+  width: 50px;
+  height: 50px;
+}
+.whats-h-icons {
+  width: 30px;
+  color: var(--accent-clr);
+  display: inline-block;
+  padding-right: 40px;
+  padding-top: 40px;
+  margin-right: 40px;
+}
+
+.left-icons-n-list .list-clr {
+  padding: 20px;
+  line-height: 60px;
+}
+.list {
+  list-style: none;
+  width: 30px;
+  height: 30px;
+  color: var(--dark-gry);
+  line-height: 60px;
+}
+.list-clr {
+  color: var(--blk-clr);
+}
+.pic-profile-1 {
+  width: 100px;
+  height: 100px;
+  padding: 20px;
+  border-radius: 50%;
+}
+.pic-profile-2 {
+  width: 70px;
+  height: 70px;
+  padding: 10px;
+  border-radius: 50%;
```

Comment:
I can see here (as well as on line 154) that you're setting the border-radius for profile pictures. You could, in future, create one css class for rounding borders, and apply that to both of the profile pictures. For instance
```css
.circle-avatar {
border-radius: 50%;
}
```

That way, you don't have to repeat yourself in the different classes you have.